translations/README.bosnian.md

```diff
@@ -0,0 +1,211 @@
+# Hasura GraphQL Engine
+
+[![Dokumenti](https://img.shields.io/badge/docs-v1.0-brightgreen.svg?style=flat)](https://docs.hasura.io)
+[![CircleCI](https://circleci.com/gh/hasura/graphql-engine.svg?style=shield)](https://circleci.com/gh/hasura/graphql-engine)
+
+
+<a href="https://discord.gg/vBPpJkS"><img src="https://img.shields.io/badge/chat-discord-brightgreen.svg?logo=discord&style=flat"></a>
+<a href="https://twitter.com/intent/follow?screen_name=HasuraHQ"><img src="https://img.shields.io/badge/Follow-HasuraHQ-blue.svg?style=flat&logo=twitter"></a>
+<a href="https://eepurl.com/dBUfJ5"><img src="https://img.shields.io/badge/newsletter-subscribe-yellow.svg?style=flat"></a>
+
+Hasura GraphQL Engine je nevjerovatno brzi GraphQL server koji vam nudi **trenutne, API-je GraphQL API-ja preko Postgres-a**, sa [**webhook pokretacima**](event-triggers.md) na desavanja u bazi podataka, i [**udaljenim semama**](remote-schemas.md) za poslovnu logiku.
+
+Hasura vam pomaze da napravite GraphQL aplikacije podrzane Postgres-om ili postepeno predjite na GraphQL za postojece aplikacije pomocu Postgres-a.
+
+Procitajte vise na [hasura.io](https://hasura.io) i [dokumentima](https://docs.hasura.io).
+
+------------------
+
+![Hasura GraphQL Engine Demo](assets/demo.gif)
```

Comment:
The images in the readme appear broken since it is inside `translations`. You might need to add `../` to the paths.